<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobMaxSettingsWidgetUI</class>
 <widget class="QGroupBox" name="JobMaxSettingsWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Max Settings</string>
  </property>
  <property name="title">
   <string>Max Settings</string>
  </property>
  <layout class="QVBoxLayout" name="mVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="m64BitCheck">
       <property name="text">
        <string>Run Max 64 Bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mPCPreviewServiceCheck">
       <property name="text">
        <string>PCPreview Service</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mRealflowRenderkitServiceCheck">
       <property name="text">
        <string>Realflow_Renderkit Service</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output Path*:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="FieldLineEdit" name="mOutputPathEdit">
       <property name="field" stdset="0">
        <string>outputPath</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mBrowseOutputPathButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel7">
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="CameraCombo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Camera01</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel8">
       <property name="text">
        <string>Height*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="FieldSpinBox" name="mHeightSpin">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="field" stdset="0">
        <string>flag_h</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel9">
       <property name="text">
        <string>Width*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="FieldSpinBox" name="mWidthSpin">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="field" stdset="0">
        <string>flag_w</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="mSetupFrameNthButton">
     <property name="text">
      <string>Setup Frame Nth Rendering...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>Flags*</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="FieldCheckBox" name="xhCheck">
        <property name="text">
         <string>Render Hidden</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="FieldCheckBox" name="xdCheck">
        <property name="text">
         <string>Displacement</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xd</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="FieldCheckBox" name="xeCheck">
        <property name="text">
         <string>Effects</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="FieldCheckBox" name="xoCheck">
        <property name="text">
         <string>Field order even</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xo</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="FieldCheckBox" name="xaCheck">
        <property name="text">
         <string>Atmospherics</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="FieldCheckBox" name="vCheck">
        <property name="text">
         <string>Video Post</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="FieldCheckBox" name="xfCheck">
        <property name="text">
         <string>Render to Fields</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xf</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="FieldCheckBox" name="x2Check">
        <property name="text">
         <string>Force 2-Sided</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_x2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="FieldCheckBox" name="xkCheck">
        <property name="text">
         <string>Super Black</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xk</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="FieldCheckBox" name="xvCheck">
        <property name="text">
         <string>Video Color Check</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xv</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="FieldCheckBox" name="xpCheck">
        <property name="text">
         <string>Paletted Output Dither</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xp</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="FieldCheckBox" name="xcCheck">
        <property name="text">
         <string>True Color Output Dither</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xc</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="FieldCheckBox" name="xnCheck">
        <property name="text">
         <string>Nth Serial Rendering</string>
        </property>
        <property name="field" stdset="0">
         <string>flag_xv</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldLineEdit</class>
   <extends>QLineEdit</extends>
   <header>fieldlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>FieldSpinBox</class>
   <extends>QSpinBox</extends>
   <header>fieldspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>FieldCheckBox</class>
   <extends>QCheckBox</extends>
   <header>fieldcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
